src/code/z_collision_check.c

```diff
@@ -3084,10 +3117,11 @@ void func_80062B80(GlobalContext* globalCtx, Vec3f* v) {
     D_8015D3D8.speed = 8.0f;
     D_8015D3D8.gravity = -1.0f;
 
-    Effect_Add(globalCtx, &sp24, EFFECT_SPARK, 0, 1, &D_8015D3D8);
+    Effect_Add(globalCtx, &pIndex, EFFECT_SPARK, 0, 1, &D_8015D3D8);
 }
 
-void func_80062CD4(GlobalContext* globalCtx, Vec3f* v) {
+// Spawns streaks of light from hits against solid objects
+void CollisionCheck_SpawnShieldParticles(GlobalContext* globalCtx, Vec3f* v) {
     static EffectShieldParticleInit init = {
         16,
         { 0.0f, 0.0f, 0.0f },
```

Comment:
outside if possible

Reply:
also can you convert the colors here to decimal

Reply:
decimal yes, outside no